web/src/app/sidebar/interested-courses/component.html

```diff
@@ -6,8 +6,10 @@
       <button type="button" class="btn btn-warning" (click)="selectedTermService.setSelectedTermAsActive()">Set current term as active</button>
     </div>
   </div>
-  <div *ngIf="showStatusText" class="sidebar-status p-2">
+  <div *ngIf="showStatusText; else hasCourses" class="sidebar-status p-2">
```

Comment:
I think `showStatusText` should be renamed to reflect it's real meaning, whether or not there are courses to display

Reply:
Hm, would "displayEmptyScheduleMessage" be apt?

Reply:
I think `isEmpty` is good